python/grass/jupyter/display.py

```diff
@@ -117,6 +128,7 @@ def run(self, module, **kwargs):
 
         # Check module is from display library then run
         if module[0] == "d":
+            self._region_manager.set_region(module, **kwargs)
```

Comment:
I don't see the implementation of the various `set_region` functions, but it seems that `set_region_from...` might be a better name to clarify what is going on.

Reply:
changed